frontend/src/features/public-form/components/FormEndPage/FormEndPage.tsx

```diff
@@ -54,6 +56,13 @@ export const FormEndPage = ({
               buttonVariant={previousSubmissionId ? 'solid' : 'outline'}
             />
           )}
+          {form?.responseMode === FormResponseMode.Multirespondent &&
+          form?.hasStatusTracker ? (
+            <SubmitAnotherResponseButton
+              endPage={endPageProps.endPage}
+              colorTheme={colorTheme}
+            />
+          ) : null}
```

Comment:
it seems like the button to "Submit Another Response" moves from above the feedback section to below the feedback section if status tracking is enabled! Just noting this is the case.